### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by wildhunter4137**

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -7,18 +7,27 @@ axios.defaults.baseURL = process.env.NEXT_PUBLIC_API;
...
+  const [messages, setMessages] = useState<number>();
 
   useEffect(() => {
-    axios.get("/statistics").then(({data}) => {
-      setStatistics(data);
-    });
+    // axios.get("http://localhost/api/statistics").then(({data}) => {
+    //   setStatistics(data);
+    // });
+    axios.get("http://localhost/api/messages/all").then(({data}) => {
```
Comment (wildhunter4137) [inline app/page.tsx:16]: You have to use the remote url

Comment (wildhunter4137) [reply app/page.tsx:16]: Sure, will do

Comment (wildhunter4137) [reply app/page.tsx:16]: Sure, will do

Comment (benjamin-h-super) [reply app/page.tsx:16]: Thank you

**Pull request merged**